### DISCUSSION TITLE
Same Syncthing ID on two Ipads

### DISCUSSION THREAD

**Discussion by jgonsior** [MobiusSync/MobiusSync#57] [Q&A] [answered]
I'm currently using two iPad devices simultaneously. I did set up the second one by transferring the data from the previous iPad. Normally one would probably get rid of the first iPad and could continue on the new one without having to set up anything for Möbius Sync again - but for my case I want to continue using both. The problem is now that both devices have the exact same syncthing device ID and appear to the whole syncthing network therefore as a single device - which they are apparently not. I don't know if now new files always will be still delivered correctly to both devices or if that may cause problems in the future, so I'm not 100% sure this is really a bug, but my intuition tells me, that both devices should have different device IDs.

**Comment by rasa** [comment]
My guess is your best bet is to delete and reinstall the app. If you'd rather not resync the files, you could use the Files app to move the files before deleting the app, and then move them back after reinstalling. Don't forget about your `.stignore` file, too.

**Comment by SimonPickup** [comment, marked as answer]
Yes, every device **must** have a different ID, so this is not a situation you want to be in. The only safe solution is to delete the app and reinstall on one of the devices.

**Comment by lmamakos** [comment]
As a suggestion, it might be nice if the application could (somehow?) detect that "transfer to new phone" thing just happened and at a minimum disabled syncing to avoid problem.  Even better would be some migration workflow to onboard the new client with a different device identifier.
